content/client-lib-development-guide/features.textile

```diff
@@ -682,18 +693,17 @@ h4. ChannelOptions
 * @(TB1)@ options provided when instancing a channel, see "Java ChannelOptions":https://github.com/ably/ably-java/blob/master/src/io/ably/types/ChannelOptions.java as a reference
 * @(TB2)@ The attributes of @ChannelOptions@ consist of:
 ** @(TB2a)@ @encrypted@ boolean - when true, enables automatic encryption & decryption of all messages
-** @(TB2b)@ @cipherParams@ @CipherParams@ - when encrypted is true, the cipher params are required
+** @(TB2b)@ @cipherParams@ - This is required if @encrypted@ is true. It consists of either:
+*** @(TB2b1)@ A @CipherParams@ instance, or
+*** @(TB2b2)@ an options hash (or language equivalent) consisting of the key @key@ with value a key. In this case, the client library should call `getDefaultParams`, passing it the key, to obtain a @CipherParams@ instance
```

Comment:
I did it that way following [@mattheworiordan's example of his preferred api](https://github.com/ably/ably-js/issues/235#issue-134905326) (presumably because that's the way it currently works in ably-ruby); happy to change to `{encrypted: true, key: ...}` if Matt and Paddy agree